pypit/scripts/coadd_1dspec.py

```diff
@@ -21,32 +21,32 @@ def parser(options=None):
     return args
 
 
-def main(args, unit_test=False, path=''):
+def main(args, unit_test=False):
     """ Runs the XSpecGui on an input file
-    path : str, optional
-      Mainly for running the unit test
     """
-    import sys
-    import pdb
     import yaml, glob
     from pypit import arcoadd
     from pypit import arspecobj
     from astropy.io import fits
 
+    if unit_test:
```

Comment:
let's just make sure this works on Travis.

Actually, I somewhat prefer my solution as
I'm pretty sure it will work dynamically.